lib/bolt/outputter/human.rb

```diff
@@ -122,7 +122,34 @@ def print_task_info(task)
         @stream.puts(task_info)
       end
 
-      def print_plan(result)
+      # @param [Hash] A hash representing the plan
+      def print_plan_info(plan)
+        # Building lots of strings...
+        pretty_params = ""
+        plan_info = ""
+        usage = "bolt plan run #{plan['name']}"
+
+        if plan['parameters']
+          plan['parameters'].each do |p|
+            name, type = p.values_at('name', 'type')
+            pretty_params << "- #{name}: #{type}\n"
+            usage << if type.to_s.include? "Optional"
```

Comment:
Optional plan params are more complex that optional task params because a plan param can be optional either because it accepts undef or because it has a default.